pkg/gpt/gpt_test.go

```diff
@@ -0,0 +1,149 @@
+package gpt
```

Comment:
These tests look very nice. Looking at `travis.sh` I do not see that we are automating testing of these in ci. Is that intentional? I think a diff like the following would be enough to ensure this was run through CI in your PR.
```diff
diff --git a/travis.sh b/travis.sh
index 01efbdd..b3f4816 100755
--- a/travis.sh
+++ b/travis.sh
@@ -37,7 +37,8 @@ echo "-----------------------> go test"
  (cd cmds && CGO_ENABLED=0 go test -a -installsuffix cgo -ldflags '-s' ./...)

 echo "-----------------------> test -cover"
- (cd cmds && CGO_ENABLED=0 go test -cover ./...)
+(cd pkg && CGO_ENABLED=0 go test -cover ./...)
+(cd cmds && CGO_ENABLED=0 go test -cover ./...)

 echo "-----------------------> go vet"
  go tool vet cmds uroot pkg
```

Reply:
done